### DISCUSSION TITLE
A Quick Look at AI in Chrome

### DISCUSSION THREAD

**Discussion by giscus** [cfjedimaster/raymondcamden2023#37] [Comments]
# A Quick Look at AI in Chrome

A look at Chrome's proposed AI integration in the web platform.

https://www.raymondcamden.com/2024/08/13/a-quick-look-at-ai-in-chrome

**Comment by Zain-ul-din** [comment]
`window.ai` 
Clever move from google they are aiming to claim AI namespace first.

**Reply by cfjedimaster** [reply]
Well to be fair, I think their intent here to create a specification that other vendors could adopt as well. If I can speak for Google. Which I can't. ;)
